website/pages/docs/reference/destination-spec.md

```diff
@@ -4,32 +4,37 @@ This goes through all the available options for the destination plugin `spec` ob
 
 ## Spec
 
-### name
+### `name`
```

Comment:
Using quotes here doesn't render well on the website (I tried it) - because they're rendered as `code`, the font is small and not the size that a header should be. I don't think it adds much to render it as code in any case, could we keep it as it was maybe?

Reply:
Good point I'l revert that change